packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -181,11 +181,44 @@ export default class BranchPublish extends Command {
       runtimeVersion = getRuntimeVersionForSDKVersion(sdkVersion);
     }
 
-    if (!runtimeVersion) {
+    let runtimeVersions: { [keyof: string]: string };
```

Comment:
this function is getting quite big, consider splitting it/extracting some parts

Reply:
```suggestion
    let runtimeVersions: Record<string, string>;
```

Reply:
Good point, extracted a bunch of this PR into a function called `getRuntimeVersionObject`.